packages/@o3r/core/schematics/ng-add/index.ts

```diff
@@ -19,7 +19,7 @@ class DevInstall extends NodePackageInstallTask {
 
   /** @inheritdoc */
   public toConfiguration() {
-    const installOptions = process.env?.npm_execpath?.includes('yarn')  ? 'yarn' : 'npm';
+    const installOptions = this.packageManager || process.env?.npm_execpath?.includes('yarn') ? 'yarn' : 'npm';
```

Comment:
Can use the getPackageManaager() from `@o3r/schematics`

Reply:
finally done in https://github.com/AmadeusITGroup/otter/pull/473